### PR TITLE
Fix error handling in Lotus

### PR DESCRIPTION
**Pull request by dnkolegov**
This PR fixes an error in error handling.

### CODE REVIEW

File: scripts/ipc/mine-subnet.sh [hunk 1]
```diff
@@ -12,4 +12,4 @@ fi
 IMG=$1
 
 echo "[*] Starting to mine on validator"
-docker exec -it $IMG eudico mir validator run --membership=onchain --nosync --ipcagent-url=http://host.docker.internal:3030/json_rpc
+docker exec -e MIR_INTERCEPTOR_OUTPUT="mir-event-logs" -it $IMG eudico mir validator run --membership=onchain --nosync --ipcagent-url=http://host.docker.internal:3030/json_rpc
```
Comment (adlrocha) [inline scripts/ipc/mine-subnet.sh:15]: Do we really want to have the interceptor enabled always? Won't this increase the size requirement a lot?

File: chain/consensus/mir/manager.go [hunk 2]
```diff
@@ -496,3 +517,44 @@ func getMembershipInfo(
...
+func validateMembershipInfo(info *mirmembership.Info) error {
+	if info == nil {
+		return fmt.Errorf("nil membership info")
+	}
+
+	if info.ValidatorSet == nil {
+		return fmt.Errorf("nil validator set in membership info")
+	}
+
+	if len(info.ValidatorSet.Validators) == 0 {
```
[the remark this answers is not in the document]
Comment (dnkolegov) [reply chain/consensus/mir/manager.go:540]: Weight of which validator? And what should we do if it is?

File: scripts/ipc/mine-subnet.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (dnkolegov) [reply scripts/ipc/mine-subnet.sh:15]: Added

Comment (dnkolegov) [reply scripts/ipc/mine-subnet.sh]: Fixed

Comment (jsoares) [reply scripts/ipc/mine-subnet.sh]: That was me; I thought the branch was just for our tests and didn't realise we would merge the thing into dev :)

**Review by jsoares — APPROVED** [review]
lgtm

**Pull request merged**